test/controllers/ChoiceControllerSpec.scala

```diff
@@ -91,15 +91,15 @@ class ChoiceControllerSpec extends CustomExportsBaseSpec with BeforeAndAfter {
         .cache[Choice](any(), ArgumentMatchers.eq(Choice.choiceId), any())(any(), any(), any())
     }
 
-    "redirect to role page when simplified declaration chosen" in {
+    "redirect to dispatch-location page when simplified declaration chosen" in {
```

Comment:
if you can please remove `-` from this place. It's test description, so you don't need to use specific naming like view or something like this. Like one problem above, it's not important to change it